module/product/src/main/kotlin/product/mapper/mapper.kt

```diff
@@ -0,0 +1,24 @@
+package product.mapper
+
+import product.dto.ProductDTO
+import product.vo.Product
+
+fun Product.toDTO(): ProductDTO {
+    return ProductDTO(
+        id = this.id,
+        name = this.name,
+        price = this.price,
+        capacity = this.capacity,
+        productStatus = this.productStatus
+    )
+}
```

Comment:
해당 로직은 엔티티 내부의 함수로 옮기는 것이 응집력 측면에서 더 좋을것같음
```suggestion
@Entity
@Table(name = "product")
data class Product(
    // ... 생략
) {
    fun toDTO () = ProductDTO(
        // ... 생략
    )
}
```